people.md

```diff
@@ -0,0 +1,60 @@
+# People in the governance structure
+
+This document lists the current people and institutions involved in
+the governance.
+
+Benevolent Dictator for Life
+============================
+
+Fernando Perez is the Benevolent Dictator for Life (BDFL).
+
+Steering Council
+================
+
```

Comment:
Emacs says the list is in alphabetical order (by first name).  M-x sort-lines....